examples/tfengine/modules/foundation.hcl

```diff
@@ -147,6 +181,9 @@ template "cicd" {
           location = "us-east1"
           name     = "cicd-pool"
         }
+        // Do not use $${module.project.project_id} for project ID.
+        service_account = "projects/{{.prefix}}-{{.env}}-devops/serviceAccounts/cloudbuild_sa"
+        logs_bucket = "gs://{{.prefix}}-logs-bucket"
```

Comment:
`logs_bucket` is mandatory when custom service account is used. We should create the bucket inside the cicd recipe and somehow enforce that dependency. Another way could be always create and use a custom logs bucket. In that case, we should check what's the implication of change in terms of the existing logs bucket.

Reply:
Done. Added requirement conditions in triggers.tf in the CICD component.